aip/general/0193.md

```diff
@@ -138,6 +138,18 @@ descriptions, and examples:
     `ErrorInfo`,`LocalizedMessage`, and so on.
   - *Example*: Described in the [ErrorDetails](#error-details) section.
 
+
+**Important:** In the context of the of the `google.rpc.Status` Protobuf
```

Comment:
Remove duplicate "of the". (And do we normally capitalize Protobuf?)

Reply:
Fixed (both).